sanity-io/structured-content-2022
# Render all Programs for new `programsSection` section

# Render all Programs in new `programsSection` section

## Intent

Render all Program entities in _/program_

## Description

This PR "breaks" `sessionsSection` sections, in that previously rendered `startTime` fields are removed, due to them being removed in Studio. However, this section is not specified to be used anywhere in the Figma sketches as far as I can see.

## Testing this PR

1. Visit [/program](https://structured-content-2022-web-gddx8n83h.sanity.build/program) and scroll down to the gray "New York City"/"Amsterdam 2022"/"Oslo 2022"/"London"/"San Francisco" accordion.
2. Verify that the content and times match the Session times in the corresponding [Program](https://stagingadmin.structuredcontent.live/desk/program) entity
